src/mpt.rs

```diff
@@ -68,10 +70,30 @@ impl MptCircuitConfig {
             &canonical_representation,
         );
 
+        // This ensures that the final mpt update in the circuit is complete, since the padding
+        // for the mpt update is a valid proof that shows the account with address 0 does not
+        // exist in an mpt with root = 0 (i.e. the mpt is empty).
+        let is_final_row = SelectorColumn(cs.fixed_column());
+        cb.add_lookup(
```

Comment:
This is a very costly solution. It is more natural and efficient with constraints:

    cb.condition(is_final_row, |cb| {
        cb.assert_equal(
        …
    });